cmake/partition_manager.cmake

```diff
@@ -10,6 +10,9 @@ define_property(GLOBAL PROPERTY PM_IMAGES
 Each image's directory will be searched for a pm.yml, and will receive a pm_config.h header file with the result.
 Also, the each image's hex file will be automatically associated with its partition.")
 
+# Create a variable which exposes the correct logical target for the current image.
+set(${IMAGE}logical_target ${logical_target_for_zephyr_elf} CACHE STRING "" FORCE)
```

Comment:
Should this be moved to boilerplate.cmake or something? I would guess it could be useful for others as well.

Reply:
When it is set here in the cache, it is available for everyone. By setting it in this file our downstream patch is smaller.